docs/devices/TH1300ZB.md

```diff
@@ -30,62 +29,62 @@ The device will restart. The wireless icon will disapear from the display.
 
 #### Occupancy / Unoccupancy
 The thermostat can be instructed to use the occupancy or unoccupancy setpoints by
-publishing to `zigbee2mqtt/FRIENDLY_NAME/set {"thermostat_occupancy": "OCCUPANCY"}`
+publishing to `zigbee2mqtt/[FRIENDLY_NAME]/set {"thermostat_occupancy": "OCCUPANCY"}`
 where `OCCUPANCY` is one of the following values: `occupied`, `unoccupied`.
 The factory default value is `occupied`.
 
 #### Floor control mode
 The sensor used by the thermostat for temperature regulation can be changed by publishing
-to `zigbee2mqtt/FRIENDLY_NAME/set {"floor_control_mode": "SENSOR"}`
+to `zigbee2mqtt/[FRIENDLY_NAME]/set {"floor_control_mode": "SENSOR"}`
```

Comment:
This part of the docs is generated, so please don't add `[` here (as it will be overwitten again).